manifests/osfamily/windows.pp

```diff
@@ -8,8 +8,12 @@
   } elsif  ($::puppet_agent::is_pe and (!$::puppet_agent::use_alternate_sources)) {
     $pe_server_version = pe_build_version()
     $tag = $::puppet_agent::arch ? {
-      'x64' => 'windows-x86_64',
-      'x86' => 'windows-i386',
+      'x64' => $::fips_enabled ? {
+          true => 'windowsfips-x64',
+          default => 'windows-x86_64' },
+      'x86' => $::fips_enabled ? {
```

Comment:
This will not work as we are not providing windowsfips x86 packages, we should let it install `windows-i386` no matter the `fips_enabled` flag